app/views/completed_transaction/show.html.erb

```diff
@@ -16,20 +15,32 @@
         <p>If you needed an organ transplant would you have one? If so please help others.</p>
         <p>If you live in Wales and want to be an organ donor, you don’t need to do anything. Find out about your choices for <a href="https://www.organdonation.nhs.uk/supporting-my-decision/welsh-legislation-what-it-means-for-me/" rel="external" class="govuk-link">organ donation in Wales</a>.</p>
         <p>
-          <%= link_to 'Join', @publication.promotion['url'],
-                title: "Register to become an organ donor", rel: "external", class: "button", role: "button" %>
+          <%= render "govuk_publishing_components/components/button", {
+            text: "Join",
```

Comment:
Outside the scope of this pull request - I'll see who I need to flag about making these more explanatory, since "Join", "Sign up", etc don't work on their own.